scico/operator/_operator.py

```diff
@@ -47,7 +49,11 @@ def _wrap_mul_div_scalar(func: Callable) -> Callable:
 
     @wraps(func)
     def wrapper(a, b):
-        if np.isscalar(b) or isinstance(b, jax.core.Tracer):
```

Comment:
I think there should be a utility function for checking if something is a SCICO scalar so we can keep the definition consistent. This can't be the only place this comes up. Would also allow us to document and test it.

Reply:
Agreed. See new function `scico.numpy.is_scalar_equiv`. (I expect you may take issue with the abbreviation in the name.)